src/app/components/concerts-upcoming/concerts-upcoming.component.html

```diff
@@ -20,15 +20,15 @@ <h2 class="primary-heading">{{ 'BVKZ.CONCERTS__COMPONENT.CONCERTS_TITLE' | trans
               target="_blank"
               [href]="concert[icon.concertUpcomingItemLineIdentifier]"
             >
-              <img class="concerts-upcoming__icon" [src]="iconsRoute + icon.iconSource" [alt]="icon.iconImageAltText"
+              <img class="concerts-upcoming__icon" [src]="icon.iconSource | iconItem" [alt]="icon.iconImageAltText"
```

Comment:
itt nem is volt svg beirva, biztos, hogy jol mukodik itt is a pipe?

Reply:
Igen, mert az icon.iconSource az egy string, ami tartalmazza a .svg-t.